client/src/components/ProTx/components/shared/plotUtils.js

```diff
@@ -315,10 +316,32 @@ const getMaltreatmentPlotData = (
  * @returns
  */
 
-const getObservedFeaturesPlotData = () => {
+const getObservedFeaturesPlotData = (
+  selectedGeographicFeature,
+  observedFeature,
+  data,
+  geography,
+  year,
+  showRate,
+  dataHistogram
+) => {
   const newObservedFeaturesPlotData = getObservedFeaturesDataObject();
+  // Add focal_value: the value for each year for the selected geographic feature
+  // eslint-disable-next-line no-restricted-syntax
+  for (const [iYear, yearData] of Object.entries(dataHistogram.years)) {
+    yearData.focal_value = getObservedFeatureValue(
+      data,
+      geography,
+      iYear,
+      observedFeature,
+      selectedGeographicFeature,
+      showRate
+    );
+  }
+  const anotherNewObservedFeaturesPlotData = dataHistogram;
```

Comment:
@taoteg , here is why  a value is created from the response.  I'm not sure if the focal_value is correct.